core/Translation/Translator.php

```diff
@@ -114,7 +114,8 @@ public function setCurrentLanguage($language)
      */
     public function getDefaultLanguage()
     {
-        return Config::getInstance()->General['default_language'];
+        $generalSection = Config::getInstance()->General;
+        return @$generalSection['default_language'] ?: 'en';
```

Comment:
Note: this change is for when global.ini.php is missing. This way, we can still use the Translator.